app/services/ems_physical_infra_dashboard_service.rb

```diff
@@ -1,5 +1,8 @@
 class EmsPhysicalInfraDashboardService < DashboardService
   include Mixins::CheckedIdMixin
+  include ActionView::Helpers::DateHelper
```

Comment:
if the `ActionView::Helpers::DateHelper` is included in the `TextualMixins::TextualRefreshStatus`, then you shouldn't need it here (unless of course something in this service directly requires the DateHelper)

Reply:
Removed it , thanks.